Cargo.toml

```diff
@@ -53,7 +53,8 @@ mut_static = "~5.0"
 anyhow = "~1.0.31"
 
 [build-dependencies]
-protobuf-codegen-pure = "~2.14"
+protobuf-codegen-pure = "~2.17"
+itertools = "0.9.0"
```

Comment:
Should this be ~0.9.0?

Reply:
That would probably be better, yes. I'll change it.